### ISSUE TITLE
Where can i find the test data set used for model builder?

### ISSUE THREAD

**Issue by tmdwns7809** [dotnet/machinelearning-modelbuilder#1207]
In the documentation from ML.net is says the following:

Model Builder splits the training data into a training set and a test set. The training data (80%) is used to train your model and the test data (20%) is held back to evaluate your model. Model Builder uses metrics to measure how good the model is.

But where can I find the used data to test my model? I cant find it

**Comment by zewditu** [comment]
Hi @tmdwns,

Yes , in the documentation it seems 20% test data can be accessible somewhere, but that is not the case ,  Model Builder split the data  into training(80%) and test (20%) in memory  and use it there by itself.  You can Evaluate your model by sample data from your dataset in evaluate step.

thanks

**Comment by torronen** [comment]
How is the data split to test and training data? The documentation is not clear if it is from the end, randomly, or based on columns that are somehow decided. This is important as improperly selected training data may cause wrong model to be selected. Is there anyway to change how the data split is done?
